packages/contentstack-import/src/utils/asset-helper.ts

```diff
@@ -247,31 +247,56 @@ export const lookupAssets = function (
   assetUids = _.uniq(assetUids);
   assetUrls = _.uniq(assetUrls);
   let entry = JSON.stringify(data.entry);
+  
+  // Note: Use this code in case production breaks
```

Comment:
@cs-raj if the production code breaks, how client is supposed to enable this code ?